### PR TITLE
[RA-1 Ch08] Update chapter08.md

### PR DESCRIPTION
**Pull request by sukhdevkapur**
This is initial pull to address issue number 375

### CODE REVIEW

**Comment by pgoyal01** [comment]
@sukhdevkapur The intent of chapter 8 is to capture Gaps, Development ideas etc. So, may I suggest that the "automated deployment" gap be made a part of, say, Section 8.2 leaving 8.1 for a general introduction/summary of the content of the rest of Chapter 8. Thanks

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 <a name="8.1"></a>
 ## 8.1 Introduction
 
+The purpose of this chapter is to identify the gaps between what is required for automated deployment of VNFs on NFVI frameworks and framework offered by OpenStack.Once gaps are identified, the next step will be to propose a plan to address these gaps. 
+The most obvious way to address the gaps will be to propose a set of APIs in the upstream OpenStack community
+
+
```
Comment (ASawwaf) [inline doc/ref_arch/openstack/chapters/chapter08.md]: I like to cover the gap between real implementation and SDO / Opensource  , So we can propose it to ETSI NFV or/and Openstack ,...etc

**Comment by rabiabdel** [comment]
@sukhdevkapur thank you so much for taking a stab at this. would you please resolve the issues raised by @pgoyal01 and @ASawwaf to get this approved and merged.

**Comment by rabiabdel** [comment]
@sukhdevkapur isn't this a duplica of #689 ? it might make sense to move the content this is proposing to #689 and close this one?

**Comment by rabiabdel** [comment]
content moved to #689 no need for two PRs.

**Pull request closed without merging**